fixtures/node_with_opa/cas/policy.dcl.license

```diff
@@ -0,0 +1,3 @@
+# SPDX-FileCopyrightText: 2020 2020 SAP SE or an SAP affiliate company and Cloud Security Client Go contributors
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Comments in dcl are possible via `//`, so we can add the license to the dcl files directly. I can fix this in a later commit.